docs/Modern-NetworkLayer.md

```diff
@@ -53,5 +53,77 @@ export default environment;
 
 Note that this is a basic example to help you get started. This example could be extended with additional features such as request/response caching (enabled e.g. when `cacheConfig.force` is false) and uploading form data for mutations (the `uploadables` parameter).
 
+## Caching
+
+Relay modern makes no assumptions about what to cache and will garbage collect any data that is no longer referenced.
+
+You have to implement your own cache strategy. A simple solution is to use `RelayQueryResponseCache`(an in-memory cache):
+
+```javascript
+import {
+  Environment,
+  Network,
+  RecordSource,
+  Store,
+} from 'relay-runtime';
+import RelayQueryResponseCache from 'relay-runtime/lib/RelayQueryResponseCache';
+
+const oneMinute = 60 * 1000;
+const cache = new RelayQueryResponseCache({ size: 250, ttl: oneMinute });
+
+function fetchQuery(
+  operation,
+  variables,
+  cacheConfig,
+) {
+  const queryID = operation.text;
+  const isMutation = operation.operationKind === 'mutation';
+  const isQuery = operation.operationKind === 'query';
+  const forceFetch = cacheConfig && cacheConfig.force;
```

Comment:
Did you experience that `cacheConfig` can be `null` or `undefined`? Because the type says it won’t https://github.com/facebook/relay/blob/e6481d7eeb7da386c907728222fbb0cbf4352ce0/packages/relay-runtime/network/RelayNetworkTypes.js#L89